explorer/app/components/Header/DrawerContent.tsx

```diff
@@ -0,0 +1,38 @@
+import { Typography } from "@mui/material";
+import React from "react";
+import { LogoProps } from "../Logo/Logo";
+import { NavLinksProps, NavLinks } from "../NavLinks/NavLinks";
+import { SocialLinksProps, SocialLinks } from "../SocialLinks/SocialLinks";
+import {
+  DrawerContainer,
+  SloganDrawerContainer,
+  SocialDrawerContainer,
+} from "./Header.styles";
+
+interface DrawerContentProps {
+  logo: LogoProps;
+  navLinks: NavLinksProps;
+  socialLinks: SocialLinksProps;
+}
+
+export const DrawerContent = ({
+  logo,
+  navLinks,
+  socialLinks,
+}: DrawerContentProps) => {
+  return (
+    <DrawerContainer>
+      {logo.slogan && (
+        <SloganDrawerContainer>
+          <Typography variant="textBody400" component="span">
+            {logo.slogan}
+          </Typography>
+        </SloganDrawerContainer>
+      )}
+      <NavLinks {...navLinks} />
+      <SocialDrawerContainer>
```

Comment:
Let's remove the `SocialDrawerContainer` wrapper - and pass in to `SocialLinks` some configurable `sx` system props that control the social link container padding and gap, as well as social link clickable target area - e.g. small/default. We can then also control colour this way too! This way, we are removing unnecessary div wrapper elements and able to configure for the variation in mobile / desktop view - or any other future use for the social links!

e.g.
```
<SocialLinks {...socialLinks} sx={{ color: "colorInk", gap: 4, px: 6, py: 2 }} />
```